### PR TITLE
Fix ms

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/qqpz2n5ma9vw5f6whmycdwn3w8

[download apk](http://filestorage.bluewallet.io:1488/1610568180.apk)

**Review by marcosrdz — APPROVED** [review]

File: screen/send/details.js [hunk 1]
```diff
@@ -973,6 +973,13 @@ export default class SendDetails extends Component {
     });
   };
 
+  setIsloading = async isLoading => {
+    const that = this;
+    return new Promise(function (resolve) {
+      that.setState({ isLoading: isLoading }, () => setTimeout(resolve, 100));
+    });
+  };
```
Comment (limpbrains) [inline screen/send/details.js:981]: Why so complicated? Or do you really need to wait for rerender to move forward? 

I would use regular BlueApp.sleep after state change

Also you probably won't need 'that = this' if you use arrow function to resolve promise

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Comment by limpbrains** [comment]
Bad bot! I didn't approve this commit! Although it's fine

**Comment by Overtorment** [comment]
Weird. I need to work on glados a bit

**Comment by marcosrdz** [comment]
It's becoming self aware

**Comment by Overtorment** [comment]
@limpbrains when you commented on PR it effectively removed your review request, as your resolution became COMMENTED. you should have requested changes.

https://github.com/BlueWallet/GlaDOS/blob/ff59f0a95cd82233968366aed14c860c511df5c4/src/main.ts#L56

**Pull request merged**